lib/changelog.rb

```diff
@@ -0,0 +1,75 @@
+class Changelog
+  include ActionView::Helpers
```

Comment:
See if you can require just `action_view/helpers/date_helper` instead of the entire gem, and then include just that as well.

```suggestion
  include ActionView::Helpers:: DateHelper
```

Reply:
@cgriego Changing the require causes this rat hole:

```
tag_helper.rb:12:in `<module:TagHelper>': uninitialized constant ActiveSupport::Concern
```

But I can change the include.

Reply:
So also add `require 'active_support/concern'` 😄